disallowed_names/depends.txt

```diff
@@ -0,0 +1 @@
+filter
```

Comment:
Please, don't use `depends.txt`, it is deprecated, use `mod.conf` instead.

Reply:
I added a mod.conf after merging but yes, depends.txt should be avoided unless you want 0.4.x compat

Reply:
CTF doesn't support 0.4-series anymore anyways. 🙂